### PR TITLE
Fix Deprecation: GeneralUtility::isAbsPath()

### LINKED ISSUE #2686: Deprecation: #95367 - GeneralUtility::isAbsPath()
Deprecation: #95367 - GeneralUtility::isAbsPath()

https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/11.5/Deprecation-95367-GeneralUtilityisAbsPath.html
.. include:: ../../Includes.txt

=================================================
Deprecation: #95367 - GeneralUtility::isAbsPath()
=================================================

See :issue:`95367`

Description
===========

The low-level TYPO3 API method
:php:`TYPO3\CMS\Core\Utility\GeneralUtility::isAbsPath()`
has been marked as deprecated.

Impact
======

Calling the method in your own PHP code will trigger a PHP :php:`E_USER_DEPRECATED` error.

Affected Installations
======================

TYPO3 installations with custom extensions calling this PHP
method are affected. You can check if you are affected via the Extension
Scanner tool provided in the Install Tool.

Migration
=========

Replace any calls to :php:`GeneralUtility::isAbsPath()` with
the exact equivalent :php:`TYPO3\CMS\Core\Utility\PathUtility::isAbsolutePath()`
which checks for the same input.

.. index:: PHP-API, FullyScanned, ext:core

### PR DESCRIPTION
**Pull request by simonschaufi**
Closes #2686

### CODE REVIEW

**Comment by simonschaufi** [comment]
coding style needs a token :cry:

**Comment by sabbelasichon** [comment]
Doesn´t matter.

**Comment by simonschaufi** [comment]
Oh, this is terribly wrong. Sorry. I will create a fix. It doesn't work that way... :man_facepalming:

File: config/v11/typo3-110.php [hunk 1]
```diff
@@ -32,11 +32,8 @@
                 StaticCallToFuncCallRector::STATIC_CALLS_TO_FUNCTIONS => ValueObjectInliner::inline([
                     new StaticCallToFuncCall('TYPO3\CMS\Core\Utility\StringUtility', 'beginsWith', 'str_starts_with'),
                     new StaticCallToFuncCall('TYPO3\CMS\Core\Utility\StringUtility', 'endsWith', 'str_ends_with'),
-                    new StaticCallToFuncCall(
-                        'TYPO3\CMS\Core\Utility\GeneralUtility',
-                        'isFirstPartOfStr',
-                        'str_starts_with'
-                    ),
+                    new StaticCallToFuncCall('TYPO3\CMS\Core\Utility\GeneralUtility', 'isFirstPartOfStr', 'str_starts_with'),
+                    new StaticCallToFuncCall('TYPO3\CMS\Core\Utility\GeneralUtility', 'isAbsPath', 'isAbsolutePath'),
```
Comment (sabbelasichon) [inline config/v11/typo3-110.php:36]: Provide a test for that.

**Pull request merged**